### PR TITLE
fix: Skip creating elasticache subnet if it is disabled

### PR DESCRIPTION
**Pull request by elainaRenee**
Skip creating the elasticache subnet if we aren't provisioning an elasticache instance

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -50,7 +50,8 @@ module "networking" {
   private_subnet_cidrs  = var.network_private_subnet_cidrs
   public_subnet_cidrs   = var.network_public_subnet_cidrs
   database_subnet_cidrs = var.network_database_subnet_cidrs
-  elasticache_subnet_cidrs = var.network_elasticache_subnet_cidrs
+  create_elasticache_subnet = var.create_elasticache
+  elasticache_subnet_cidrs  = var.network_elasticache_subnet_cidrs
```
Comment (jsbroks) [inline main.tf:54]: did you run `terraform fmt -recursive`

**Review by jsbroks — APPROVED** [review]
lgtm

**Review by vanpelt — COMMENTED** [review]
I wonder if it would be simpler to just attach elasticache to the existing private_subnet.  Do you recall there being a specific reason for an additional subnet?

**Comment by jsbroks** [comment]
This PR is included in version 1.0.1 :tada:

**Pull request merged**